packages/bruno-app/src/providers/ReduxStore/slices/collections/actions.js

```diff
@@ -295,19 +289,10 @@ export const renameItem = (newName, itemUid, collectionUid) => (dispatch, getSta
     }
     const { ipcRenderer } = window;
 
-    ipcRenderer
-      .invoke('renderer:rename-item', item.pathname, newPathname, newName)
-      .then(() => {
-        // In case of Mac and Linux, we get the unlinkDir and addDir IPC events from electron which takes care of updating the state
-        // But in windows we don't get those events, so we need to update the state manually
-        // This looks like an issue in our watcher library chokidar
-        // GH: https://github.com/usebruno/bruno/issues/251
-        if (isWindowsOS()) {
-          dispatch(_renameItem({ newName, itemUid, collectionUid }));
-        }
-        resolve();
-      })
-      .catch(reject);
+    ipcRenderer.invoke('renderer:rename-item', item.pathname, newPathname, newName).then(() => {
+      dispatch(_renameItem({ newName, itemUid, collectionUid }))
```

Comment:
I think the `_renameItem` call can be removed, because this was basically the original fix.

Reply:
Removed